src/util/log.cpp

```diff
@@ -4,68 +4,68 @@
 #include <cstdlib>
 #include <cstdarg>
 #include <stdexcept>
-#include <execinfo.h>
+//#include <execinfo.h>
 #include <cxxabi.h>
 #include <unistd.h>
 
 #define STREAM stderr
 
 // Kinda borrowed from https://panthema.net/2008/0901-stacktrace-demangled/
 void print_stacktrace(unsigned int max_frames) {
-    void* addrlist[max_frames+1];
+    //void* addrlist[max_frames+1];
 
-    int addrlen = backtrace(addrlist, sizeof(addrlist) / sizeof(void*));
+    //int addrlen = backtrace(addrlist, sizeof(addrlist) / sizeof(void*));
```

Comment:
This whole file needs to be conditional (or maybe just not compiled on windows at all and conditional in the header).